src/pages/paymentMethods/index.jsx

```diff
@@ -0,0 +1,158 @@
+import Slider from "react-slick";
+
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import Layout from "@/layout/Layout";
+import PaymentCard from "@/components/Cards/PaymentCard";
+import Button from "@/components/elements/Button";
+import { useEffect, useState } from "react";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useTranslation } from "next-i18next";
+import { useRouter } from "next/router";
+import Link from "next/link";
+
+export default function PaymentMethods() {
+    const { t } = useTranslation("common");
+    const router = useRouter();
+    const language = router.locale;
+
+    useEffect(() => {
+        document.body.dir = language == "ar" ? "rtl" : "ltr";
+    }, [language]);
+
+    const [cards, setCards] = useState([
+        {
```

Comment:
Hey @Lilicod , for this array of objects i think it's better to assigne it to variable in the util folder inside the library file and import it here to use it in the useEffect, thank you.

Reply:
Great work @Lilicod well done.